bin/runex

```diff
@@ -22,4 +22,4 @@ if command -v accumulo > /dev/null 2>&1 ; then
   av_arg="-Daccumulo.version=`accumulo version`"
 fi
 
-mvn -q exec:java -Dexec.mainClass="org.apache.accumulo.examples.$main_class" $av_arg -Dexec.args="$main_args"
+mvn -q exec:java -Dexec.mainClass="org.apache.accumulo.examples.$main_class" $av_arg -Dexec.args="$main_args" -Dlog4j.configuration=file:"./src/main/resources/log4j.properties"
```

Comment:
Not sure this part is necessary, since the default is to use the `log4j.properties` from the class path. Putting the file in `src/main/resources` should be sufficient to get it on the classpath.

The above will also fail if the `runex` command is not executed from the root of the project as `bin/runex`. From any other directory, the relative path won't work.